### PR TITLE
move buttons into dropdown button and reopen course

### PR DESCRIPTION
**Pull request by yongsokheng**

### CODE REVIEW

File: app/models/course.rb [hunk 1]
```diff
@@ -68,6 +68,12 @@ def finish_course current_user
     create_activity key: "course.finish_course", owner: current_user
   end
 
+  def reopen_course current_user
+    self.update_attributes status: :init
+    self.user_subjects.update_all(status: Course.statuses[:init])
```
Comment (truonglocbinh) [inline app/models/course.rb]: remove ()

File: app/views/admin/subjects/_subject_info.html.erb [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+          class="btn btn-default">
+          <span class="glyphicon glyphicon-align-justify"
+            aria-hidden="true"></span>
+        </button>
+        <ul class="dropdown-menu dropdown-menu-right"
+          aria-labelledby="dLabel">
+          <li>
+            <% if @course.progress? %>
+              <% if @user_subjects.init.any? %>
+                  <a href="<%= admin_status_subject_path(@course_subject, "start") %>",
```
Comment (truonglocbinh) [inline app/views/admin/subjects/_subject_info.html.erb]: indent

File: app/views/admin/courses/show.html.erb [hunk 3]
```diff
@@ -51,6 +37,30 @@
             <ul class="dropdown-menu dropdown-menu-right"
               aria-labelledby="dLabel">
               <li>
+                <% if @course.init? %>
+                  <a href="<%= admin_course_change_status_courses_path(@course) %>",
```
Comment (truonglocbinh) [inline app/views/admin/courses/show.html.erb]: remove ()

File: app/views/admin/subjects/_subject_info.html.erb [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+                  <%= t "admin.actions.show_subject.finish_all" %>
+                </a>
+              <% end %>
+            <% end %>
+          </li>
+          <li>
+            <a href="<%= edit_admin_course_course_subject_path @course,
+              @course_subject %>">
+              <i class="fa fa-pencil" aria-hidden="true"></i>
+              <%= t"courses.buttons.admin.edit" %>
```
Comment (truonglocbinh) [inline app/views/admin/subjects/_subject_info.html.erb]: t "courses.buttons.admin.edit"

File: app/views/admin/courses/show.html.erb [hunk 5]
```diff
@@ -51,6 +37,38 @@
...
+                  data-method="post",
+                  data-confirm="<%= t "courses.confirms.confirm_clone" %>">
+                  <i class="fa fa-clone" aria-hidden="true"></i>
+                  <%= t "button.clone" %>
+                </a>
+              </li>
+              <li>
                 <a  href="<%= edit_admin_course_path %>">
                   <i class="fa fa-pencil" aria-hidden="true"></i>
                   <%= t"courses.buttons.admin.edit" %>
```
Comment (truonglocbinh) [inline app/views/admin/courses/show.html.erb]: t "" space

File: app/views/admin/subjects/_subject_info.html.erb [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+                  data-method="patch",
+                  data-remote="true",
+                  data-confirm="<%= t "admin.confirms.start_all" %>">
+                  <i class="fa fa-check" aria-hidden="true"></i>
+                  <%= t "admin.actions.show_subject.start_all" %>
+                </a>
+              <% end %>
+
+              <% unless @user_subjects.size == @user_subjects.finish.size %>
+                <a href="<%= admin_status_subject_path(@course_subject, "finish") %>",
```
Comment (truonglocbinh) [inline app/views/admin/subjects/_subject_info.html.erb]: remove ()

File: app/views/admin/subjects/_subject_info.html.erb [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+          class="btn btn-default">
+          <span class="glyphicon glyphicon-align-justify"
+            aria-hidden="true"></span>
+        </button>
+        <ul class="dropdown-menu dropdown-menu-right"
+          aria-labelledby="dLabel">
+          <li>
+            <% if @course.progress? %>
+              <% if @user_subjects.init.any? %>
+                <a href="<%= admin_status_subject_path(@course_subject, "start") %>",
```
Comment (truonglocbinh) [inline app/views/admin/subjects/_subject_info.html.erb]: remove ()

**Comment by vuducluan** [comment]
OK

**Comment by truonglocbinh** [comment]
ok

**Pull request merged**